.github/workflows/tox_verify.yaml

```diff
@@ -146,9 +144,6 @@ jobs:
     strategy:
       matrix:
         include:
-          # Still getting requests.exceptions.ReadTimeout: UnixHTTPConnectionPool() in CI, even with local EELS
```

Comment:
Note the comment regarding the fix required for #1787 :skull:

Reply:
3.11/3.12 is just so much faster that the resolver didn't timeout? lol. Hard to believe, but I don't have a better suggestion :)

From https://docs.python.org/3/whatsnew/3.11.html#summary-release-highlights:
> Python 3.11 is between 10-60% faster than Python 3.10. On average, we measured a 1.25x speedup on the standard benchmark suite. See [Faster CPython](https://docs.python.org/3/whatsnew/3.11.html#whatsnew311-faster-cpython) for details.

Reply:
Unless I'm misunderstanding, #1787 was a different issue :)